### ISSUE TITLE
Drawer should apply 'aria-expanded' attribute

### ISSUE THREAD

**Issue by cahirodoherty-learningpool** [adaptlearning/adapt-contrib-core#409]
### Subject of the issue/enhancement/features
According to success criteria 4.1.2 Name, Role, Value. Buttons and links that unfold / collapse content on the page are to be coded with aria-expanded attribute.

### Your environment
* FW5.31.12

### Expected behaviour
Expanded drawer should have 'aria-expanded' attribute applied. Collapsing the drawer should remove this

### Actual behaviour
No such attribute exists
